code/modules/mob/mob.dm

```diff
@@ -1252,13 +1252,21 @@ Use this proc preferably at the end of an equipment loadout
 	A.examine(src)
 
 
-/mob/living/verb/verb_pickup(obj/I in view(1))
+/mob/living/verb/verb_pickup(obj/I in objects_in_view(usr, 1))
 	set name = "Pick up"
 	set category = "Object"
 
 	face_atom(I)
 	I.verb_pickup(src)
 
+/proc/objects_in_view(var/mob/living/L, var/range)
```

Comment:
this is a questionable name for a proc that filters by `can_pickup`

Reply:
that's fair, I'll rename it

interactable_objects_in_view?

objects_in_view_acquirable?